keyboards/kingly_keys/ave/readme.md

```diff
@@ -0,0 +1,23 @@
+# The Ave. (Avenue)
+
+![ave-img-1](https://i.imgur.com/l5ah6uM.png)
```

Comment:
General note: this render shows a layout that isn't supported by the codebase.

Reply:
The 2u uses the same key as the full grid so royal did not see a reason to add the layout

Reply:
Yeah what @Spaceman said.

But I can change it to another if you would prefer

Reply:
Eh, I'll allow it. I'm sure I'm nitpicking, but I find these things and then they bother me.